contribute/contribute-pull-requests/add_hook.md

```diff
@@ -0,0 +1,14 @@
+# How To add a new Hook
+Adding a new hook is quite a simple process. It can be done in only tree steps.
```

Comment:
```suggestion
Adding a new hook is a process that can be done in only three steps.
```

We should refrain from using "simple", "easy", etc., in the documentation. It might not be that simple for some :)